lib_src/wscript

```diff
@@ -0,0 +1,11 @@
+def use_module(bld):
+    bld.env.append_value('XCC_FLAGS', ['-Wno-missing-braces', '-O3'])
```

Comment:
Should this now be setting `MODULE_XCC_FLAGS` with the contents of `XCC_FLAGS` plus the library specified flags?

Reply:
Yes, this did need changing. I've updated it now.